src/Aquifer.API/Endpoints/Resources/Content/SendForPublisherReview/Community/Endpoint.cs

```diff
@@ -0,0 +1,60 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Resources.Content.SendForPublisherReview.Community;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService, IResourceHistoryService historyService) 
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Post("/resources/content/{ContentId}/send-for-publisher-review-community");
+        Permissions(PermissionName.SendReviewCommunityContent);
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var user = await userService.GetUserFromJwtAsync(ct);
+        var contentVersion = await dbContext.ResourceContentVersions
+            .Where(x => x.ResourceContentId == request.ContentId && x.AssignedUserId == user.Id && x.IsDraft)
+            .Include(x => x.ResourceContent)
+            .SingleAsync(ct);
+
+        if (contentVersion is null)
+        {
+            ThrowError("Content not found");
```

Comment:
This will return a 500 which isn't really right. I would `await SendNotFoundAsync(); return;` to return a 404 which implies nothing was found.